### PR TITLE
RDoc-1523 - Proofreading

### PR DESCRIPTION
**Pull request by morhilai**
Fixed grammar and spelling errors; also adjusted some formatting for consistency and clarity.

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 ## What is CSV
 
-A comma-separated values (CSV) file is a delimited text file that uses a comma to separate values ( [from wikipedia](https://en.wikipedia.org/wiki/Comma-separated_values)).
+*A Comma-Separated Values (CSV) file is a delimited text file that uses a comma to separate values* ([from Wikipedia](https://en.wikipedia.org/wiki/Comma-separated_values)).
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: Verify the usage of the '*' here 

Did you want the whole sentence to appear italics ?
-or - 
Did you just want a bullet before ?

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown [hunk 2]
```diff
@@ -49,16 +49,16 @@ NestedObject.Name
 Inner Object
 {CODE-BLOCK/}
 
-The import process will combine properties with the same prefix back into a JSON object. 
+The import process will combine properties with the same prefix back into one JSON object.  
 
-The "ArrayObject" property is an array and as such contains multiple values and should be escaped as string like so: 
+The `ArrayObject` property is an array and as such contains multiple values. These should be escaped as strings like so:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: strings => a string

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown [hunk 3]
```diff
@@ -49,16 +49,16 @@ NestedObject.Name
...
-The "ArrayObject" property is an array and as such contains multiple values and should be escaped as string like so: 
+The `ArrayObject` property is an array and as such contains multiple values. These should be escaped as strings like so:  
 
 {CODE-BLOCK:json}
 ArrayObject
 "[1,2,3,4]"
 {CODE-BLOCK/}
 
-Last thing we need inorder for an import to work is to add the `collection` property otherwise the name of the collection will derive from the CSV file name.
+Last thing we need in order for an import to work is to add the `collection` property. If we don't, the name of the collection will derive from the CSV file name.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: will derive => will be derived

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown [hunk 4]
```diff
@@ -72,15 +72,14 @@ If we want to import the document with a specific `id` we need to include an `@i
 Samples/1-A
 {CODE-BLOCK/} 
 
-The end result should look like so:
+The end result should look like this:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: =>
The complete CSV line should look like this:

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown [hunk 5]
```diff
@@ -72,15 +72,14 @@ If we want to import the document with a specific `id` we need to include an `@i
...
 Samples/1-A,Import from CSV,Inner Object,"[1,2,3,4]",Samples
 {CODE-BLOCK/} 
 
-Now that we got a valid CSV file we can import it to RavenDB by selecting  
-[`Target database`]->[`Settings`]->[`Import Data`]->[`From CSV file`] like so:  
-![Figure 1. Import CSV file](images/csv-import.JPG "Import CSV file")
+Now that we've got a valid CSV file we can import it to RavenDB by selecting the target database and navigating to  
+`Settings`>`Import Data`>`From CSV file`  
 
-The end result should look like the document above
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: =>
After importing the CSV file, the resulting document will look like the document [above](....)

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown [hunk 6]
```diff
@@ -1,41 +1,45 @@
...
+- **Include Conflicts**: determines whether Conflicts contained in the file should be imported.  
+- **Include Indexes**: determines whether Indexes contained in the file should be imported.  
+- **Remove Analyzers**: determines whether the Analyzers used by indexes should be scripted.  
+- **Include Identities**: determines whether Identities contained in the file should be imported.  
+- **Include Configuration**: determines whether configurations of Revisions, Expiration and Client contained in the file should be imported.  
+- **Include Compare Exchange**: determines whether Compare Exchange values contained in the file should be imported.  
 
 {NOTE:Note}
-If any of the options is set but the file doesn't contain any items of that type it will not error. 
+If any of the options are set but the file doesn't contain any items of that type, it will not throw an error.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown]: => No error is thrown if you select an option that is Not contained in the file

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown [hunk 7]
```diff
@@ -1,41 +1,45 @@
...
 
 {NOTE:Note}
-If any of the options is set but the file doesn't contain any items of that type it will not error. 
+If any of the options are set but the file doesn't contain any items of that type, it will not throw an error. 
 {NOTE/}
 
-## Advanced import options
-![Figure 1. Advanced import options](images/advanced-import-options.JPG "Advanced import options")
+## Advanced Import Options
+![Figure 1. Advanced import options](images/advanced-import-options.JPG)
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown]: Return original

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown (hunk 6 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown (hunk 7 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-data-file.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown (hunk 5 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown (hunk 4 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/studio/database/tasks/import-data/import-from-csv.markdown]: Done

**Pull request merged**